code/transportation-development-services/tds.js

```diff
@@ -614,6 +627,253 @@ $(document).on('knack-view-render.view_744', function(event, view, record) {
+          ${dropdownMenuItem(recordId, "memo-builder", "fa-medium", "Memo Builder")}\
+          ${dropdownMenuItem(recordId, "tia-case-log", "fa-briefcase", "Case Log")}\
+        </ul>\
+      </li>\
+    </ul>\
+  </div>`).appendTo("#view_893")
+})
+
+/* Scope Submission Details Page */
+$(document).on('knack-view-render.view_901', function(event, view, record) {
+  // regex: match the 24 digit record id that comes in hash part of url after "/"
+  const parentRecordId = window.location.hash.match(/(?<=\/)\d[a-z0-9]{23}/)[0];
+  const recordId = view.scene.scene_id;
+
+  $(`<div class="details-dropdown-menu tabs">\
+    <ul id="tia-menu-list">\
+      <li class="tia-dropdown-menu kn-dropdown-menu kn-button">\
+        <a href="#tia-requests/tia-case-details/${recordId}/tia-case-management/${recordId}" data-kn-slug="#case-management">\
+          <span class="nav-dropdown-link">Case Management</span>\
+          <span class="kn-dropdown-icon fa fa-caret-down" />\
+        </a>\
+        <ul class="kn-dropdown-menu-list tia-dropdown-menu-list" style="min-width: 152px; margin: 0;">\
+          ${dropdownMenuItem(recordId, "tia-case-details", "fa-list-alt", "Case Details")}\
+          ${dropdownMenuItem(recordId, "tia-case-management", "fa-archive", "Scope & Submissions")}\
+          ${dropdownMenuItem(recordId, "tia-mitigation-details", "fa-file-text-o", "Mitigations")}\
+          ${dropdownMenuItem(recordId, "memo-builder", "fa-medium", "Memo Builder")}\
+          ${dropdownMenuItem(recordId, "tia-case-log", "fa-briefcase", "Case Log")}\
+        </ul>\
+      </li>\
+      ${dropdownMenuItem(recordId, "assign-scope-submission-reviewer", "fa-hand-o-up", "Assign Reviewer")}\
+      ${dropdownMenuItem(recordId, "begin-scope-submission-review", "fa-play-circle-o", "Begin Review")}\
+      ${dropdownMenuItem(recordId, "complete-scope-submission-review", "fa-exchange", "Reject/Approve")}\
+      ${dropdownMenuItem(recordId, "scope-submission-change-log", "fa-list-ol", "Submission Log")}\
+    </ul>\
+  </div>`).appendTo("#view_901")
+})
+
+/* Submission Details Page */
+$(document).on('knack-view-render.view_902', function(event, view, record) {
+  // regex: match the 24 digit record id that comes in hash part of url after "/"
+  const parentRecordId = window.location.hash.match(/(?<=\/)\d[a-z0-9]{23}/)[0]
+  const recordId = view.scene.scene_id;
+
+  $(`<div class="details-dropdown-menu tabs">\
+    <ul id="tia-menu-list">\
+      <li class="tia-dropdown-menu kn-dropdown-menu kn-button">\
+        <a href="#tia-requests/tia-case-details/${recordId}/tia-case-management/${recordId}" data-kn-slug="#case-management">\
+          <span class="nav-dropdown-link">Case Management</span>\
+          <span class="kn-dropdown-icon fa fa-caret-down" />\
+        </a>\
+        <ul class="kn-dropdown-menu-list tia-dropdown-menu-list" style="min-width: 152px; margin: 0;">\
```

Comment:
These will also need to be updated to parentRecordId for both.

`${dropdownMenuItem(parentRecordId, "tia-case-details", "fa-list-alt", "Case Details")}\
          ${dropdownMenuItem(parentRecordId, "tia-case-management", "fa-archive", "Scope & Submissions")}\
          ${dropdownMenuItem(parentRecordId, "tia-mitigation-details", "fa-file-text-o", "Mitigations")}\
          ${dropdownMenuItem(parentRecordId, "tia-memo-details", "fa-medium", "Memo Details")}\
          ${dropdownMenuItem(parentRecordId, "tia-case-log", "fa-briefcase", "Case Log")}\`